ibm/service/power/data_source_ibm_pi_keys_test.go

```diff
@@ -31,6 +31,5 @@ func testAccCheckIBMPIKeysDataSourceConfig() string {
 	return fmt.Sprintf(`
 		data "ibm_pi_keys" "test" {
 			pi_cloud_instance_id = "%s"
-		}
-	`, acc.Pi_cloud_instance_id)
+		} `, acc.Pi_cloud_instance_id)
```

Comment:
There's an extra space between "}" and "`" that needs to be removed on this line.

Reply:
changes made